website/docs/tutorials/react.md

```diff
@@ -245,13 +245,13 @@ Catalog statistics:
 That's great! So, how we're going to load it into your app? [LinguiJS](https://github.com/lingui/js-lingui) introduces concept of compiled message catalogs. Before we load messages into our app, we need to compile them. As you see in the help in command output, we use [`compile`](/docs/ref/cli.md#compile) for that:
 
 ```bash
-> lingui compile
+> lingui compile --typescript
```

Comment:
Let's leave this example without a parameter by default, and add a tip at the bottom (as we did before for the CLI article).